rpc/params/apierror.go

```diff
@@ -192,7 +173,6 @@ const (
 	CodeAlreadyExists             = "already exists"
 	CodeUpgradeInProgress         = "upgrade in progress"
 	CodeMigrationInProgress       = "model migration in progress"
-	CodeIncompatibleClient        = "incompatible client"
```

Comment:
The only reason I can see keeping this around is because clients might get this depending on what controller they interact with. If we are doing manual QA can we confirm that a 3.1.7 cli client gets a reasonable error from an old controller that doesn't allow it to negotiate a facade version?